python/src/ContinuousPC_doc.i.in

```diff
@@ -1,59 +1,100 @@
 %feature("docstring") OTAGRUM::ContinuousPC
-"ContinuousPC learner.
+"ContinuousPC learner
```

Comment:
@MLasserre you can keep the dots at the end, see https://www.python.org/dev/peps/pep-0257/:
`The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".`

Reply:
OK, I will fix that for when I push more doc.

Reply:
so should I wait before merging ?

Reply:
Yes,  if you don't mind.